qcow2reader_test.go

```diff
@@ -1,26 +1,22 @@
-package qcow2reader
+// package qcow2reader_test keeps blackbox tests for qcow2reader.
```

Comment:
nit
```suggestion
// Package qcow2reader_test keeps blackbox tests for qcow2reader.
```

https://tip.golang.org/doc/comment

Reply:
Fixed